src/app/components/CallToActionLinkComponent/Text/index.tsx

```diff
@@ -0,0 +1,43 @@
+/** @jsx jsx */
+
+import { PropsWithChildren } from 'react';
+import { jsx } from '@emotion/react';
+import TEXT from '#app/components/Text';
+import { FontVariant, GelFontSize } from '../../../models/types/theming';
+import styles from './index.styles';
+
+type TextProps = {
+  as?: string;
+  fontVariant?: FontVariant;
+  size?: GelFontSize;
+  className?: string;
+  overrideInteractionStyles?: boolean;
```

Comment:
Can we perhaps give this a more descriptive name? What does it mean? Shall we call it `shouldUnderlineOnHoverFocus` or something along those lines?